### PR TITLE
Quiet moab rep audit

### PR DESCRIPTION
**Pull request by tallenaz**

### CODE REVIEW

File: app/jobs/moab_replication_audit_job.rb [hunk 1]
```diff
@@ -20,7 +20,7 @@ def perform(complete_moab)
       check_aws_replicated_zipped_moab_version(zmv, results)
     end
 
-    results.report_results(logger)
+    # results.report_results(logger)
```
Comment (ndushay) [inline app/jobs/moab_replication_audit_job.rb]: could you add a comment explaining why you are commenting this out?

File: app/services/audit_results.rb [hunk 2]
```diff
@@ -97,12 +97,12 @@ class AuditResults
   ].freeze
 
   HONEYBADGER_REPORT_CODES = [
```
Comment (ndushay) [inline app/services/audit_results.rb]: similarly, would it be helpful to add a comment explaining why you are commenting this out (I don't have enough context to know how these codes are used)

File: app/jobs/moab_replication_audit_job.rb (hunk 1 above)
Comment (jmartin-sul) [inline app/jobs/moab_replication_audit_job.rb]: i think i'd somewhat prefer to leave this line as-is...

File: app/services/audit_results.rb [hunk 3]
```diff
@@ -97,12 +97,12 @@ class AuditResults
   ].freeze
```
Comment (jmartin-sul) [inline app/services/audit_results.rb]: and would instead suggest commenting the replication related result codes in this list, similar to what's done w/ the HB codes.

alternatively, we could just comment out `results.report_results(logger)` (as you've done above), and leave the reporting codes in this class alone entirely.  i think i have a slight preference for commenting the result codes... but mostly i'd want to just comment result codes, or just comment the reporting invocation, but not half-n-half.

**Comment by tallenaz** [comment]
@ndushay are the changes you requested OK?

**Pull request merged**